src/pages/workspaces/workspace/analysis/tool-utils.ts

```diff
@@ -0,0 +1,134 @@
+import _ from 'lodash/fp'
+import { isCromwellAppVisible } from 'src/libs/config'
+import * as Utils from 'src/libs/utils'
+import { Extension, getExtension } from 'src/pages/workspaces/workspace/analysis/file-utils'
+
+
+export type RuntimeToolLabel = 'Jupyter' | 'RStudio' | 'Azure'
+export type AppToolLabel = 'Galaxy' | 'Cromwell'
+export type MiscToolLabel = 'spark' | 'terminal'
+export type ToolLabel = RuntimeToolLabel | AppToolLabel | MiscToolLabel
+
+export const toolLabelTypes: Record<ToolLabel, ToolLabel> = {
+  Jupyter: 'Jupyter',
+  RStudio: 'RStudio',
+  terminal: 'terminal',
+  spark: 'spark',
+  Azure: 'Azure',
+  Galaxy: 'Galaxy',
+  Cromwell: 'Cromwell'
+}
+
+export const appToolLabelTypes: Record<AppToolLabel, AppToolLabel> = {
+  Galaxy: 'Galaxy',
+  Cromwell: 'Cromwell'
+}
+
+export const isToolLabelAnApp = (x: ToolLabel): x is AppToolLabel => x in appToolLabelTypes
```

Comment:
Naming nit: I think another name might read a bit more clearly. As is, this reads like "Is this tool label an app?". To me, it seems the answer to that question is always no: a label isn't an app.

Maybe one of `isAppToolLabel` or `isToolAnApp`?

Reply:
I like how `isAppToolLabel` reads